generator.yml

```diff
@@ -245,6 +245,21 @@ builds:
   - {<< : *x86_64_cut,    << : *android-4_14,     << : *lld,             boot: true,  llvm_version: *llvm_tot}
   - {<< : *arm64_cut,     << : *android-4_9,      << : *clang,           boot: true,  llvm_version: *llvm_tot}
   - {<< : *x86_64_cut,    << : *android-4_9,      << : *clang,           boot: true,  llvm_version: *llvm_tot}
+  #############
+  #  LTO/CFI  #
+  #############
+  - {<< : *arm64,        << : *lto-cfi,          << : *llvm_full,        boot: true,  llvm_version: *llvm_tot}
+  - {<< : *arm64_lto,    << : *lto-cfi,          << : *llvm_full,        boot: true,  llvm_version: *llvm_tot}
```

Comment:
for example, when I see this anchor, I think "full LTO or ThinLTO" then have to go look at the anchor. not a big deal though.

Reply:
Ah, I see. Fair point although I am not sure full LTO is worth testing through this matrix. Something to double back on if it is relevant in the future.